ios/FluentUI/List/ListItem.swift

```diff
@@ -0,0 +1,154 @@
+                    let subtitleView = Text(subtitle)
+                        .foregroundColor(Color(uiColor: tokenSet[.subtitleColor].uiColor))
+                        .lineLimit(state.subtitleLineLimit)
+                        .truncationMode(state.subtitleTruncationMode)
+                    if footer.isEmpty {
+                        subtitleView
+                            .font(Font(tokenSet[.subtitleTwoLinesFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.subtitleTwoLineHeight)
+                    } else {
+                        subtitleView
+                            .font(Font(tokenSet[.subtitleThreeLinesFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.subtitleThreeLineHeight)
+                        Text(footer)
+                            .foregroundColor(Color(uiColor: tokenSet[.footerColor].uiColor))
+                            .font(Font(tokenSet[.footerFont].uiFont))
+                            .frame(minHeight: ListItemTokenSet.footerHeight)
+                            .lineLimit(state.footerLineLimit)
+                            .truncationMode(state.footerTruncationMode)
+                    }
+                }
+            }
+        }
+
+        @ViewBuilder
+        var accessoryView: some View {
+            if let accessoryType = state.accessoryType,
+               let icon = accessoryType.icon,
+               let iconColor = accessoryType.iconColor(tokenSet: tokenSet, fluentTheme: fluentTheme) {
+                let image = Image(uiImage: icon)
+                    .foregroundColor(Color(uiColor: iconColor))
+                if accessoryType == .detailButton {
+                    SwiftUI.Button {
+                        if let onAccessoryTapped = state.onAccessoryTapped {
+                            onAccessoryTapped()
+                        }
+                    } label: {
+                        image
+                    }
+                } else {
+                    image
+                }
+            }
+        }
+
+        @ViewBuilder
+        var backgroundView: some View {
+            if let backgroundColor = state.backgroundStyleType.defaultColor(tokenSet: tokenSet) {
+                Color(backgroundColor)
+            }
+            EmptyView()
+        }
+
+        @ViewBuilder
+        var contentView: some View {
+            HStack(alignment: .center) {
+                leadingView()
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                labelsView
+                    .padding(.trailing, ListItemTokenSet.horizontalSpacing)
+                Spacer()
+                trailingView()
+                    .tint(Color(fluentTheme.color(.brandForeground1)))
+                accessoryView
+                    .padding(.leading, ListItemTokenSet.horizontalSpacing)
+            }
+            .padding(EdgeInsets(top: ListItemTokenSet.paddingVertical + 1,
```

Comment:
@huwilkes and I were discussing this. I am unsure of what to do for this scenario as the padding in the SwiftUI control should be 12 according to the design. But the token for the TableViewCell is 11 and it isnt super clear why yet.

So for now I did this not so ideal work around.

Reply:
IMO this has higher probability of causing a hard to find future bug once we forget about this +1. Probably better to just follow what the token has.